src/components/RadioGroup/examples/default.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import RadioGroup from '../RadioGroup';
+
+const style = {
+	marginRight: '13px'
```

Comment:
i'm not sure? should we have styles in our docs less which are for specific examples?